build.gradle.kts

```diff
@@ -55,22 +74,21 @@ configure(projects) {
         implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:$kotlinSerializationVersion")
         implementation("org.jetbrains.kotlinx:kotlinx-serialization-protobuf:$kotlinSerializationVersion")
 
-
         // Dependency Injection
-        implementation("org.kodein.di:kodein-di-conf-jvm:7.7.0")
+        implementation("org.kodein.di:kodein-di-conf-jvm:7.8.0")
 
         // Logging
-        implementation("org.slf4j:slf4j-api:1.7.30")
-        implementation("ch.qos.logback:logback-classic:1.2.3")
-        implementation("io.github.microutils:kotlin-logging:2.0.6")
+        implementation("org.slf4j:slf4j-api:1.7.32")
+        implementation("ch.qos.logback:logback-classic:1.2.6")
+        implementation("io.github.microutils:kotlin-logging:2.0.11")
 
         // ReactiveX
         implementation("io.reactivex:rxjava:1.3.8")
         implementation("io.reactivex:rxkotlin:1.0.0")
         implementation("com.jakewharton.rxrelay:rxrelay:1.2.0")
 
         // dependency both in AndroidCompat and extensions, version locked by Tachiyomi app/extensions
-        implementation("org.jsoup:jsoup:1.14.1")
+        implementation("org.jsoup:jsoup:1.14.2")
```

Comment:
This is version locked by extensions, is it fine to use a newer version?

Reply:
I believe Tachi has updated this in the app, so it should be fine